root/components/common-macros.tt

```diff
@@ -249,7 +243,7 @@ END -%]
       ELSIF entity.isa('MusicBrainz::Server::Entity::URL'); link_url(entity, action, text);
       END;
     ELSIF entity.isa('MusicBrainz::Server::Entity::URL');
-      simple_link(url_escape(entity.url), display_url(entity.url));
+      simple_link(entity.href_url, entity.pretty_name);
```

Comment:
This was MBS-8413, the double-encoding issue. Using `url_escape` is practically always wrong (I’ll remove it altogether in another part).